mindmeld/components/entity_resolver.py

```diff
@@ -275,84 +493,21 @@ def fit(self, clean=False):
                 use_double_metaphone=self._use_double_metaphone,
             )
 
-        self.ready = True
```

Comment:
why is this op removed?

Reply:
The op is moved to `fit()` method rather than modifying that in derived class' `_fit()` method.